server/views/case-summary-record-order.njk

```diff
@@ -317,75 +317,92 @@
+                    </dd>
                 {% endfor %}
 
-                    <dt class="govuk-heading-m govuk-!-margin-bottom-1 pac-dashboard-count">{{ absentAppointments.complied | length }}</dt>
-                    <dd class="govuk-body govuk-!-font-weight-bold govuk-!-margin-left-0 govuk-!-margin-bottom-0 govuk-!-display-inline-block">Absences</dd>
+                <dd class="govuk-!-margin-left-0 govuk-!-margin-top-1">
+                    <span class="govuk-heading-m govuk-!-margin-bottom-1 pac-dashboard-count">{{ absentAppointments.complied | length }}</span>
+                    <span class="govuk-body govuk-!-font-weight-bold govuk-!-margin-left-0 govuk-!-margin-bottom-0 govuk-!-display-inline-block">Absences</span>
+                </dd>
 
                 {% for complied in absentComplied | sort(true, false, 'count') %}
-                    <div class="govuk-body govuk-!-margin-0">
-                        <dt class="govuk-body pac-dashboard-count">{{ complied.count }}</dt>
-                        <dd class="govuk-body govuk-!-margin-left-0 govuk-!-margin-bottom-0 govuk-!-display-inline-block">{{ complied.description }}</dd>
-                    </div>
+                    <dd class="govuk-body govuk-!-margin-0">
+                        <span class="govuk-body pac-dashboard-count">{{ complied.count }}</span>
+                        <span class="govuk-body govuk-!-margin-left-0 govuk-!-margin-bottom-0 govuk-!-display-inline-block">{{ complied.description }}</span>
+                    </dd>
                 {% endfor %}
-            </div>
-        </dl>
+            </dl>
+        </div>
 
 
         <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible govuk-!-width-two-thirds">
 
 
-        <dl class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third pac-!-grid-column-18 govuk-heading-m govuk-!-margin-bottom-2">
-                    <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">Unacceptable</h3>
-                </div>
-
-                <div class="govuk-grid-column-two-thirds govuk-!-margin-top-0 govuk-heading-m govuk-!-margin-bottom-1">
-                    <dt class="pac-dashboard-count">{{ attendedAppointments.failed | length }}</dt>
-                    <dd class="govuk-body govuk-!-font-weight-bold govuk-!-margin-left-0 govuk-!-margin-bottom-0 govuk-!-display-inline-block">Attendances</dd>
+        <div class="govuk-grid-row">
+            <dl class="govuk-grid-column-one-third pac-!-grid-column-18 govuk-!-margin-top-0">
+                <dt class="govuk-heading-m govuk-!-margin-bottom-0">
+                    <h3 class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block"
+                        aria-hidden="true">Unacceptable</h3>
+                </dt>
+                <dd hidden="true">Unacceptable</dd>
```

Comment:
Same as previous, don't think we need this line and the dl/dt tags can be divs/spans